docs/fsharp/language-reference/slices.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Slices (F#)
+description: Learn about how to use slices for existing F# data types and how to define your own slices for other data types.
+ms.date: 01/22/2019
+---
+
+# Slices
+
+In F#, a slice is a subset of a data type. To be able to take a slice from a data type, some implementation of the `GetSlice` method must be implemented either in the data type or in a [type extension](type-extensions.md) that is in scope. This article will explain how to take slices from existing F# types and how to define your own.
```

Comment:
- some implementation...must be implemented is awkward. Perhaps, "...either the data type or a type extension that is in scope must provide an implementation of the GetSlice method."
- will explain --> explains